sources/packages/api/src/route-controllers/config/config.controller.ts

```diff
@@ -16,6 +16,7 @@ export class ConfigController {
         url: config.auth.url,
         realm: config.auth.realm,
         clientIds: config.auth.clientIds,
+        externalLogoutUrl: config.auth.externalLogoutUrl,
```

Comment:
Maybe we should make the variable name explicit about what externalLogout it is, for instance, externalSiteMinderLogoutUrl.